docs/k0s-single-node.md

```diff
@@ -0,0 +1,44 @@
+# k0s Single Node Quick Start
+This outlines a quick method to running local k0s master and worker in one node. 
+
+  1) Prepare Dependencies
+      1. Download the k0s binary
+      ```sh
+       sudo curl --output /usr/local/sbin/k0s -L https://github.com/k0sproject/k0s/releases/download/v0.7.0/k0s-v0.7.0-amd64
+      ```
+      2. Download the kubectl binary
+      ```sh
+       sudo curl --output /usr/local/sbin/kubectl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+      ```
+      3. Make both binaries executeable
+      ```sh
+       sudo chmod +x /usr/local/sbin/{kubectl,k0s}
+      ```
+
+  2) Start k0s
+      1. Create ~/.k0s directory
+      ```sh
+       mkdir -p ${HOME}/.k0s
+      ```
+      2. Generate default k0s.yaml cluster configuration
+      ```sh
+       k0s default-config | tee ${HOME}/.k0s/k0s.yaml
+      ```
+      3. Start k0s
+      ```sh
+       sudo k0s server -c ${HOME}/k0s.yaml --enable-worker &
```

Comment:
I think this way to run k0s is not stable because after server restarting k0s won't be running. Add a note about systemd/supervisord etc.

Reply:
I'm not sure that creating a systemd unit file belongs to a quick start guide

Reply:
Maybe to note this in footnotes?

Reply:
Or a "Further reading" section.

Reply:
I think it makes sense to show how to make k0s persistent via systemd or supervisord but that fits in this doc better as a reference link.

It might be better to create an FRE to track codifying a k0s sub command to instantiate that task in a way that gives you as developers the control over how it is practiced.

Reply:
We already have it planned https://github.com/k0sproject/k0s/issues/206
Stay tuned.